templates/types/streaming/express/src/controllers/llamaindex-stream.ts

```diff
@@ -50,12 +50,12 @@ export const convertMessageContent = (
     });
   }
 
-  if (additionalData?.uploadedCsv) {
+  if (additionalData?.csvFiles?.length) {
+    const rawContents = additionalData.csvFiles.map((csv) => {
+      return "```csv\n" + csv.content + "\n```";
```

Comment:
Including file name or file path to the raw content is not a bad idea. How do you think @thucpn ?